app/footer.tsx

```diff
@@ -1,3 +1,176 @@
+        <blockquote className="items flex flex-col items-center justify-center px-10">
+          <Image
+            src="https://s3-alpha-sig.figma.com/img/18b7/a13d/8bbb852e070e69b3de2a5ac59d20f501?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=gtaYqKeU8izdKAOskR5FvCKz15ZM9I~Fai49G9uVnIM1GLBs1e0sYvodH7rpjGkPIJ2yOczgOM6i~jECdaO3ufZHCUY~~80goh86AFkw~6vyJmKTmFVNUlz5kHfqMFmuaPqjTuoJY7XeFeDdYHTjrvGGNe6ATas9-IYPC2gSKavFZ8L5~tNY1vn-~IDnjdeemmKPOsyWbKDmUJWTfI~s~wsWLnKiwVFYpPjYZtwj-u1-~Dy2SuSFOKALF-AnZk1oSBIJrwnSS~Ox6aAP64YsclxQDO6NyJGWBbZP3s5jgVh6tHrHSzR9irkt9nyZj0ndSptaXYTRx4QFx8MYRfeDdw__"
+            alt="NIT KURUKSHETRA LOGO"
+            className="pt-5"
+            width={66}
+            height={66}
+          />
+          <header className="font-display max-w-[250px] pt-3 text-center font-serif text-[22px] text-[#fff]">
+            National Institute of Technology, Kurukshetra
+          </header>
+          <aside className="max-w-[250px] pt-3 text-center font-sans text-[16px] text-[#8B949E]">
+            WRXF+23G, NIT, Thanesar, Haryana 136119
+          </aside>
+
+          <object>
+            <ul className="flex flex-row items-center justify-center pt-3">
+              <li className="pr-5">
+                <a href="">
+                  <MdOutlineCall style={{ color: 'white', fontSize: '22px' }} />
+                </a>
+              </li>
+              <li className="pr-5">
+                <a href="">
+                  <MdMailOutline style={{ color: 'white', fontSize: '22px' }} />
+                </a>
+              </li>
+              <li className="pr-5">
+                <a href="">
+                  <CiGlobe style={{ color: 'white', fontSize: '22px' }} />
+                </a>
+              </li>
+              <li className="pr-5">
+                <a href="">
+                  <SlLocationPin style={{ color: 'white', fontSize: '22px' }} />
+                </a>
+              </li>
+            </ul>
+          </object>
+        </blockquote>
+
+        <figure className="flex origin-center rotate-90 items-center justify-center lg:rotate-0">
+          <Image
+            width={210}
+            height={350}
+            src="https://s3-alpha-sig.figma.com/img/aeae/a206/68c4e501ecf5c3911343c1f3d8428cdf?Expires=1707696000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=S22f7ZBENQn-7VFFtfnHencMUHU6jvAsYS9gcgivdkyDW2~TCXcFb9ByJoCACBymHLXh50v7hy7SEOi-yEJYm2Xvk~yqKrUMtfCkWRjfh9EDErVFYx7nMQ4kQOSFTqSQWeUs9qymatl-CMJV8nlc-GvQ~bG8Xlij24qfYl~L9mGzSqSWUtPUi6xcmRKcLYbVMdisnKbn5NsDrpZ8ASxHyfBzarMRHULKzDvQEgmep9BoCkaOY-5UwOJF9~Z-Puo~5lVBh3olPUJQud689q3h095Sd9Yv0lZ9HnzpesX0V689O5wCrPen5nSY2lOhL69pKpP4Ixk~ynkZtdRIZFskoQ__"
+            className="h-[300px]"
+            alt="Design Glyph"
+            layout="fixed"
+          />
+        </figure>
+
+        <footer className="grid grid-cols-2 py-5 pl-5 lg:grid-cols-3">
+          <menu className="px-5 pb-5 lg:px-10">
+            <ul>
```

Comment:
If you bother to read the docs, `menu` is a semantic _replacement_ for `ul`. You've nested both here.